### ISSUE TITLE
Demo page broken

### ISSUE THREAD

**Issue by neothemachine** [kartena/Leaflet.Pancontrol#14]
Tested with latest Firefox.

**Comment by perliedman** [comment]
Which version is that, and on which platform?

Just tried it on Firefox 38/Linux Ubuntu 14.04, and it seems to work as expected.

**Comment by neothemachine** [comment]
Sorry, I found out it's only broken when I access it via https (to which 
the HTTPS Everywhere browser extension switches to automatically):
https://kartena.github.io/Leaflet.Pancontrol/

**Comment by perliedman** [comment]
Yeah, the page loads the Leaflet JS and CSS from http, which is blocked if the page is loaded with https; I guess we could switch to using protocol relative URLs for loading them.

**Comment by simison** [comment]
@perliedman see https://github.com/kartena/Leaflet.Pancontrol/pull/15 ;-) Does just that.

**Comment by perliedman** [comment]
#15 fixes the demo page, but not the `gh-pages` branch, so the public demo is still broken. Easy to fix, but since I'm on vacation and with a very limited connection, I will let it be for now.

**Comment by timwaters** [comment]
I see it has been merged now.

It would be great to change the About website link to: https://kartena.github.io/Leaflet.Pancontrol/
